Jenkinsfile

```diff
@@ -0,0 +1,19 @@
+node('lisk-explorer-01'){
+  stage ('Prepare Workspace') {
+    deleteDir()
+    checkout scm
+  }
+
+  stage ('Shellcheck') {
+    try {
+      sh '''#!/bin/bash -xe
+      # shellcheck
+      shopt -s globstar; /opt/shellcheck/shellcheck **/*.sh
```

Comment:
Can we make sure shellcheck is in the $PATH so that we do not have to hard code its absolute path here?